recipes-multimedia/imx-codec/imx-codec_4.9.0.bb

```diff
@@ -60,7 +59,7 @@ do_package_qa[prefuncs] += "__set_insane_skip"
 
 python __split_libfslcodec_plugins() {
     codecdir = bb.data.expand('${libdir}', d)
-    do_split_packages(d, codecdir, r'^lib_([^_]*).*_arm.*_elinux\.so\..*',
+    do_split_packages(d, codecdir, '^lib_([^_]*).*_arm.*_elinux\.so\..*',
```

Comment:
This is the wrong way. The line in `meta-freescale` needs to be backported to `meta-imx`.